app-hw-37/src/handleAuth.ts

```diff
@@ -0,0 +1,63 @@
+import React, { ReactNode } from 'react';
+const AUTHURL = "https://studapi.teachmeskills.by/auth/users/"
```

Comment:
reasons why it's good to save domain and enpoints separately 
- easy to migrate
- tests
- different domains for different environments